pkg/plugin/plugin.go

```diff
@@ -92,7 +92,7 @@ func ParseVersion(version string) (v Version, err error) {
 	}
 
 	// Parse stage suffix, if any.
-	v.Stage = strings.TrimPrefix(submatches[3], "-")
```

Comment:
Why not trim the spaces?

Reply:
We are timming the dash (`-`), not the spaces, and by modifying the regex as I did the dash is not part of the capture group now, so no need to trim it.